README.md

```diff
@@ -1,10 +1,72 @@
+
+**[WHY]** An OSPO serves as a vital bridge between an organization and the open source community, helping to ensure that the organization is a good steward of open source software and can reap the benefits of open source adoption while minimizing risks.
+
+**[WHO]** OSPOs are composed of people (open source specialists) wearing different hats:
+
+* Open Source Enabler: OSPOs can help organizations navigate the cultural, process, and tool changes required to engage with the open source community effectively. This can involve educating teams/ units, establishing new processes and workflows, and adopting new tools and technologies.
+
+* Open Source Counselor: OSPOs can provide guidance and advice on the latest open source trends, licensing issues, and how to engage with open source projects, foundations, and communities. This can help organizations stay up-to-date with the rapidly changing open source landscape and ensure they are making informed decisions.
+
+* Open Source Advocate: OSPOs can promote the use and/or contribution of open source and best practices across different organizational units. This can help organizations realize the benefits of open source as well as engaging people to contribute to open source projects or start new ones.
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+**[HOW]** The way the people behind an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. The OSPO's strategy focuses on aligning the organization's open source goals with its overall organization objectives.
+
+> OSPOs can be formed in various sectors, regions, and organizational sizes including private and public organizations such as academics, NGOs, foundations, governments, and public administrations, as well as small, medium, and large companies. It's also important that there are many different names for an OSPO-like structure within an organization. The naming and concrete organizational form will strongly depend on the the practices of each organization. Organizing it as a "Program" or an "Office" is only one way to implement a structure which serves the purpose of an OSPO.
+
+## Basic Segmentation
```

Comment:
The main reason for this is that categorizations can help to better understand and reference the different OSPOs created by looking at their motivators. For instance, there are certain topics relevant to corporate OSPOs that are completely out of the scope of other OSPOs created within universities or governments, and vice versa. Having a set of basic categories for people to use can help to differentiate and find similarities in OSPOs being established across sectors.

To give all of you context, this comes from a past issue discussion and work on categories started in OSPOlogy repo: https://github.com/todogroup/ospology/issues/261 and was also implemented in the ospo-book project in the taxonomy section, reviewed by contributors of the project: https://github.com/todogroup/ospology/blob/main/ospo-book/chapters/taxonomy.md

Reply:
Following up on this @lhawthorn 🙂  Since the review period is already past due and this comment is the only one missing, I'll add in this section a disclaimer mentioning that the *basic segmentation* is in *beta version*. This way, we can focus on your comment and work on this section to decide whether or not can be improved or should be deleted in the future. It can also be a good chance to bring more people from other OSPO communities with deep experience in other angles (gov, NGOs) to provide input.